{"event": "verbose", "uuid": "5c98324a-ed67-43d7-aa84-f674861ec70e", "counter": 3, "stdout": "\u001b[0;31mThe error appears to be in '/home/vescana/test_ansible_molecule/playbooks/site.yml': line 24, column 7, but may\u001b[0m", "start_line": 2, "end_line": 3, "runner_ident": "76d35e48-7f21-4abe-b745-72c5e7c06807", "created": "2022-04-11T13:11:33.250066"}